ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/aaa_authorization.schema.yml

```
@@ -0,0 +1,31 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  aaa_authorization:
+    type: dict
```

Please always use display name for acronyms
```suggestion
    type: dict
    display_name: AAA Authorization
```